src/screens/Inventory/CreatePurchaseRequest.tsx

```diff
@@ -39,19 +39,20 @@ function CreatePurchaseRequest(props: CreatePurchaseRequestProps) {
   const { classes } = props;
   const history = useHistory();
   const userId = useSelector(userIdSelector);
-  const inventory = useSelector(currentInventorySelector);
-  const product = useSelector(currentInventoryProductSelector);
```

Comment:
Would you mind doing some redirecting if someone navigates to this URL by typing it in (maybe just redirecting them to inventoryMain) when these aren't set yet? I'll be doing the equivalent handling in the customer-related pages.